certified-connectors/InEight/apiDefinition.swagger.json

```diff
@@ -1645,47 +1654,50 @@
             "in": "body",
             "required": false,
             "schema": {
-              "type": "object",
-              "properties": {
-                "DailyPlanId": {
-                  "type": "integer",
-                  "format": "int32",
-                  "description": "DailyPlanId"
-                },
-                "PlanStatusCode": {
-                  "type": "string",
-                  "description": "PlanStatusCode"
-                },
-                "ModifiedById": {
-                  "type": "string",
-                  "description": "ModifiedById"
-                },
-                "ModifiedDate": {
-                  "type": "string",
-                  "description": "ModifiedDate"
-                },
-                "DtModifiedDate": {
-                  "type": "string",
-                  "description": "DtModifiedDate"
-                },
-                "ReleasedDate": {
-                  "type": "string",
-                  "description": "ReleasedDate"
-                },
-                "DtReleasedDate": {
-                  "type": "string",
-                  "description": "DtReleasedDate"
-                },
-                "ReleasedById": {
-                  "type": "string",
-                  "description": "ReleasedById"
-                },
-                "Errors": {
-                  "type": "array",
-                  "items": {
-                    "type": "string"
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "DailyPlanId": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "DailyPlanId"
+                  },
+                  "PlanStatusCode": {
+                    "type": "string",
+                    "description": "PlanStatusCode"
                   },
-                  "description": "Errors"
+                  "ModifiedById": {
+                    "type": "string",
+                    "description": "ModifiedById"
+                  },
+                  "ModifiedDate": {
+                    "type": "string",
+                    "description": "ModifiedDate"
+                  },
+                  "DtModifiedDate": {
+                    "type": "string",
+                    "description": "DtModifiedDate"
+                  },
+                  "ReleasedDate": {
+                    "type": "string",
+                    "description": "ReleasedDate"
+                  },
+                  "DtReleasedDate": {
+                    "type": "string",
+                    "description": "DtReleasedDate"
```

Comment:
@InEightDevOps All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Updated description as per the comments